### DISCUSSION TITLE
Ottai CGM / Companion APP

### DISCUSSION THREAD

**Discussion by dchukhutin** [NightscoutFoundation/xDrip#3738] [General]
Would it be possible to read out the Ottai CGM using the Companion App function?
Ottai sends notifications with glucose values ​​every 5 minutes, but xDrip does not see them. The data source selected in xDrip is Companion APP.
Smartphone - Samsung Galaxy A54
![image](https://github.com/user-attachments/assets/6ff9e87d-aae4-47ae-bb2c-ecffb7f578a0)
![image](https://github.com/user-attachments/assets/574b4d36-058a-48cb-9907-be490e01c180)

**Comment by Navid200** [comment]
Please try the latest release: 2024.10.29

**Reply by dchukhutin** [reply]
It works. Thank you very much!!!
